service/controller/resource/nodeautorepair/metrics.go

```diff
@@ -0,0 +1,32 @@
+package nodeautorepair
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	gaugeValue float64 = 1
+)
+
+var (
+	nodeAutoRepairTermination = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "aws_operator_node_auto_repair_termination",
+			Help: "Metric representing node termination due node auto repair feature.",
```

Comment:
```suggestion
			Help: "Gauge representing node termination due to node auto repair feature.",
```

BTW, why gauge and not counter? This number can only go up, so I think a counter would fit better.

Reply:
It will never have other value than 1

Reply:
each instance is only terminated once